change/@uifabric-example-app-base-2019-08-01-23-28-00-visuals.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Should probably be patch
```suggestion
  "type": "patch",
```